docs/docs/target/container_image.md

```diff
@@ -381,59 +161,48 @@ Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
 </details>
 
 ### OCI Layout
-Trivy supports image directories compliant with [Open Container Image Layout Specification](https://github.com/opencontainers/image-spec/blob/master/spec.md).
+Trivy supports scanning image filesystem directories compliant with [Open Container Image Layout Specification](https://github.com/opencontainers/image-spec/blob/master/spec.md).
+
+Examples:
 
 Buildah:
 
-```
+```bash
 $ buildah push docker.io/library/alpine:3.11 oci:/path/to/alpine
 $ trivy image --input /path/to/alpine
 ```
 
 Skopeo:
 
-```
+```bash
 $ skopeo copy docker-daemon:alpine:3.11 oci:/path/to/alpine
 $ trivy image --input /path/to/alpine
 ```
 
 Referencing specific images can be done by their tag or by their manifest digest:
-```
+
+```bash
 # Referenced by tag
 $ trivy image --input /path/to/alpine:3.15
 
 # Referenced by digest
 $ trivy image --input /path/to/alpine@sha256:82389ea44e50c696aba18393b168a833929506f5b29b9d75eb817acceb6d54ba
 ```
 
-## SBOM
-Trivy supports the generation of Software Bill of Materials (SBOM) for container images and the search for SBOMs during vulnerability scanning.
-
-### Generation
-Trivy can generate SBOM for container images.
-See [here](../supply-chain/sbom.md) for details.
+### Complementing SBOM
+Any SBOM file found in the container image will be scanned for vulnerabilities and contribute to the image's final scaning report. This is useful if you're building container images without package manager information (i.e distroless) but you want to ensure the could be scannable in the future.
 
-### Discover SBOM inside container images
-Trivy can search for Software Bill of Materials (SBOMs) within container image files and scan their components for vulnerabilities.
+### Substituting SBOM
+Trivy can avoid analyzing the image contents (which can be time and resource consuming task), and instead scan and SBOM and was pre-generated for the scanned image. This can speed up and simplify repeated scans of the same image.
```

Comment:
PTAL at this section which I rewrote after the rebase. did I get it right that sbom in the image is added to the result but sbom in external source is replacing the result?

Reply:
ref: https://github.com/aquasecurity/trivy/discussions/8547

Reply:
@DmitriyLewen can you PTAL?

Reply:
You are right.
We don't check image, if SBOM from external source found:
https://github.com/aquasecurity/trivy/blob/529957eac1fc790c57fa3d93524a901ce842a9f5/pkg/fanal/artifact/image/image.go#L113-L115